OptimizelySDK.NetStandard16/OptimizelySDK.NetStandard16.csproj

```diff
@@ -3,7 +3,6 @@
     <PropertyGroup>
         <TargetFramework>netstandard1.6</TargetFramework>
         <ReleaseVersion>1.2.1</ReleaseVersion>
-        <SignAssembly>true</SignAssembly>
```

Comment:
are we no longer signing the distributions?

Reply:
We're still signing the distributions, but the process we follow is changed.
To sn.exe (typically we only do this during prep of a *.nupkg), we either check on 4 Options checkboxes in VS on our local Mac machine or run a *.ps1 script.  The strong naming and digital signing we put in place for our *.nupkg releases remains and is still QA checked that it happens.

Reply:
The change is necessitated by E2E which would otherwise generate "CSC : error CS7027: Error signing output with public key from file" errors.  If it sounds a bit familiar, it is because it an issue we've been wrestling with and perfecting.